### ISSUE TITLE
Missing panels from row in mobile view

### ISSUE THREAD

**Issue by GVLTodorov** [grafana/grafana#39037]
**What happened**:
Video MP4
https://github.com/GVLTodorov/public/raw/main/bug/Grafana%20Bug%202021-09-09%2015-10-17%20(online-video-cutter.com).mp4

**What you expected to happen**:
Mobile browser should show all panels in row when expanded.

**How to reproduce it (as minimally and precisely as possible)**:
row by default collapsed and saved.
with chrome browser from desktop [simulating mobile device] and mobile device.

**Environment**:
- Grafana version: v8.1.3 (a61f38238c)
- Data source type & version: InfluxDB 2.0.8
- OS Grafana is installed on: Ubuntu 20.04 with container image: grafana/grafana:latest
- User OS & Browser: Windows 10 Pro 21H1, Chrome 93.0.4577.63
- Grafana plugins: grafana-clock-panel,briangann-gauge-panel,natel-plotly-panel,grafana-simple-json-datasource,cloudspout-button-panel
- Others:

**Comment by dprokop** [comment]
@GVLTodorov thanks for reporting this isssue. Similar ones already reported:
- https://github.com/grafana/grafana/issues/16978 - fixed in v 6.2, so this is either a regression or some edge case
- https://github.com/grafana/grafana/issues/38549

I've tried to reproduce this issue with the following dashboard https://play.grafana.org/d/V118GdSnk/rows-mobile?orgId=1. Unfortunately - no success reproducing. I've tried with device emulation in chrome and on Pixel 3, Chrome 93.0.4577.62
